.gitignore

```diff
@@ -28,3 +28,7 @@ SolarTherm/Resources
 cscope.*
 *.deb
 wheel
+SolarTherm/Systems/Private
```

Comment:
wondering if these folder names should appear here in the public version of solartherm?